data/data/manifests/openshift/baremetal-provisioning-config.yaml.template

```diff
@@ -9,4 +9,5 @@ spec:
   provisioningNetwork: "{{.Baremetal.ProvisioningNetwork}}"
   provisioningDHCPRange: "{{.Baremetal.ProvisioningDHCPRange}}"
   provisioningOSDownloadURL: "{{.ProvisioningOSDownloadURL}}"
+  additionalNTPServers: "{{.Baremetal.AdditionalNTPServers}}"
```

Comment:
Needs to be converted to a comma seperated list, similar to what you did in ironic.env.template

Reply:
will need to be a list to match whats expected in the CBO PR
https://github.com/openshift/cluster-baremetal-operator/pull/449/files#diff-3199d649a76f2244e3cbe100488dc1ecab6e87dbca7efb6842da31ee63505f7aR152

Reply:
Done